contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/statsbeat_metrics/statsbeat.py

```diff
@@ -52,8 +53,17 @@ def _get_stats_short_export_interval():
         return _DEFAULT_STATS_SHORT_EXPORT_INTERVAL
 
 
+def _get_stats_long_export_interval():
+    ei_env = os.environ.get("APPLICATION_INSIGHTS_STATS_LONG_EXPORT_INTERVAL")
```

Comment:
Having interval and CS as env variables could cause unexpected behaviors if configured incorrectly, do we really need to make these configurable?

Reply:
These were suggested by @heyams to be used for testing. We will not have this in our documentation for the customer to use.